modules/appeals_api/app/swagger/appeals_api/v2/swagger.json

```diff
@@ -1489,10 +1489,6 @@
                       "type": "string",
                       "pattern": "[^ \\f\\n\\r\\t\\v\\u00a0\\u1680\\u2000-\\u200a\\u2028\\u2029\\u202f\\u205f\\u3000\\ufeff]"
                     },
```

Comment:
These changes to `swagger.json` and `swagger_dev.json` probably shouldn't be here, but they seem to be introduced even if I run the existing `rswag:appeals_api:run` task on the master branch without any of these other changes. I'm not sure what I'm missing - these differences seem to have something to do with API-16004, but I'm not clear on why what I'm generating with rake doesn't match what's in the repo - does this seem like a familiar problem?

Reply:
It's somewhat familiar, yeah - we often have to be careful about what gets included into our doc files when generated. The reason most of these are happening is because we've made changes to our schema files to use the shared schemas that are still waiting to go live on production, but are used for doc generation.

It shouldn't be generating files outside of it's scope (:run/:prod should be producing `swagger.json` while :dev should be producing `swagger_dev.json`), so that is something we should look into!

Reply:
Here's what I'm gathering:
* The sources for these docs have changed in a way that does not match what's in production, so we haven't run the generation task for them
* This PR should not commit new changes to swagger.json or swagger_dev.json for that reason

Is that right?

> It shouldn't be generating files outside of it's scope (:run/:prod should be producing swagger.json while :dev should be producing swagger_dev.json), so that is something we should look into! 

I'm not sure what this means - the `rswag:appeals_api:run` and `rswag:appeals_api:dev` tasks still generate the same `swagger.json` and `swagger_dev.json` files as before. Happy to revise the structure of the new tasks however is convenient though.

Reply:
> * The sources for these docs have changed in a way that does not match what's in production, so we haven't run the generation task for them
Correct
> * This PR should not commit new changes to swagger.json or swagger_dev.json for that reason
Correct

> I'm not sure what this means - the rswag:appeals_api:run and rswag:appeals_api:dev tasks still generate the same swagger.json and swagger_dev.json files as before. Happy to revise the structure of the new tasks however is convenient though.
I was mentioning this because I couldn't tell whether you'd run both `:prod` and `:dev` to generate both swagger files, or if you'd only e.g. run `:prod` but still gotten the `swagger_dev.json` file generated. I just want to make sure we maintain that running `:prod` only produces `*.json` files and `:dev` produces `*_dev.json`. That seems to still be the case so we're good!